### PR TITLE
Pivot override styles

### LINKED ISSUE #4390: Can't override Pivot and PivotItem styles with styled-components
#### Describe the issue:

This is actually a repost of https://github.com/OfficeDev/office-ui-fabric-core/issues/997

It seems that I have the same issue as described there.

I will copy the text of the original issue description here:

Apparently there is no a direct way to override the styles of these components. There is no a prop for styling and also the className prop is completely ignored. I'm using React.

Any suggestions?

### PR DESCRIPTION
**Pull request by natalieethell**
#### Pull request checklist

- [x] Addresses an existing issue: Fixes #4390 
- [x] Include a change request file using `$ npm run change`

#### Description of changes

Add optional className to Pivot props to allow customization of component styles.

### CODE REVIEW

File: packages/office-ui-fabric-react/src/components/Pivot/Pivot.tsx [hunk 1]
```diff
@@ -105,7 +105,7 @@ export class Pivot extends BaseComponent<IPivotProps, IPivotState> {
     return (
       <FocusZone direction={ FocusZoneDirection.horizontal }>
         <ul
-          className={ css('ms-Pivot', styles.root,
+          className={ css('ms-Pivot', styles.root, this.props.className,
```
[the remark this answers is not in the document]
Comment (dzearing) [reply packages/office-ui-fabric-react/src/components/Pivot/Pivot.tsx]: don't want this to go stale. :)

[the remark this answers is not in the document]
Comment (natalieethell) [reply packages/office-ui-fabric-react/src/components/Pivot/Pivot.tsx]: Yes! So sorry about the delay. Just pushed the changes. Let me know what you think.

[the remark this answers is not in the document]
Comment (natalieethell) [reply packages/office-ui-fabric-react/src/components/Pivot/Pivot.tsx]: Working on the merge conflict.

File: packages/office-ui-fabric-react/src/components/Pivot/Pivot.base.tsx [hunk 2]
```diff
@@ -101,8 +103,10 @@ export class PivotBase extends BaseComponent<IPivotProps, IPivotState> {
   }
 
   public render(): JSX.Element {
+    const divProps = getNativeProps(this.props, divProperties);
+
     return (
-      <div>
+      <div { ...divProps } >
```
Comment (dzearing) [inline packages/office-ui-fabric-react/src/components/Pivot/Pivot.base.tsx]: nit, space at end.

Also update tests :)

**Comment by Chaitra1006** [comment]
Any updates on how to override the Pivot styles?
Reference to issue https://github.com/OfficeDev/office-ui-fabric-react/issues/4390

**Comment by JasonGore** [comment]
You should now be able to use the [IPivotStyles interface](http://localhost:4322/#/examples/pivot) to define a function or an object for the `styles` prop, or use Theming. Here is a [CodePen example](https://codepen.io/jasongore/pen/YvoRgZ) showing each approach.

**Pull request merged**